tock-process-console/src/ui_management/pages/setup_page/setup_page.rs

```diff
@@ -59,12 +101,43 @@ impl Component for SetupPage {
     where
         Self: Sized,
     {
+        let available_ports = match tokio_serial::available_ports() {
+            Ok(ports) => ports,
+            Err(error) => panic!("ports not found! : {}", error),
```

Comment:
It's not that htere are no ports found, but there was an IO error in listing available ports. Perhaps the message could reflect that better